### DISCUSSION TITLE
ISCE unwrapping

### DISCUSSION THREAD

**Discussion by nadiahassanpour** [isce-framework/isce2#648] [General]
when I process sentinel-1 data with isce, my unwrapping step with snaphu is too long because 3 bursts are processed. Is there a way to crop my area before unwrapping?

**Comment by fdelgadodelapuente** [comment]
Hi nadiahassanpour,

You can do several things.

- Take 19 looks in range and 5 looks in azimuth for a pixel size of ~75 x 75 m2 . That way snaphu should unwrap your interferogram in a few minutes.
- Use icu instead of snaphu. icu is much faster but doesn't work well in areas of low coherence.
- Increase the filtering strength.

Francisco

**Reply by nadiahassanpour** [reply]
Hi Francisco, Thanks for your suggestions. My work is such that I have to output without multi-looking (1*1).

**Reply by nadiahassanpour** [reply]
One more question, I am working in an urban area with high coherence, what filter strength do you think is appropriate?

**Reply by Daffodiles** [reply]
@nadiahassanpour  Hello, have you solved it？I have the same problem.

**Comment by fdelgadodelapuente** [comment]
Hi nadiahassanpour,

I assume you're doing PS-like analysis? If that's the case then you should not filter the data. If your data is full resolution and unfiltered, then it is likely snaphu will not run successfully.

Francisco

**Reply by nadiahassanpour** [reply]
Yes, I want to use isce to mintpy in full resolution for an urban area. Can I use icu instead of snaphu?

**Comment by fdelgadodelapuente** [comment]
You can but icu is more likely to fail on unfiltered full res data.

**Reply by HN-Jiang** [reply]
Is that possible to crop the interferogram instead of multi look? I'm in the same situation. I need 1:1 look to get more point in a very small area.
